wiki/Twitter/en.md

```diff
@@ -16,6 +16,7 @@ osu! has several Twitter accounts, each serving specific purposes. Most are very
 | ![osu! avatar](img/osugame.jpg) | [@osugame](https://twitter.com/osugame) | The official source of news and announcements. |
 | ![BanchoBoat avatar](img/banchoboat.jpg) | [@banchoboat](https://twitter.com/banchoboat) | Comedy relief when things go wrong. |
 | ![NAT avatar](img/osu_nat.png) | [@osu_nat](https://twitter.com/osu_nat) | News, announcements, and short community surveys by the [NAT](/wiki/People/The_Team/Nomination_Assessment_Team) (not run by [the osu! team](/wiki/People/The_Team)). |
+| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Difficulty calculation announcements for all rulesets, run by the [Performance Points Committee](/wiki/People/Performance_Points_Committee). |
```

Comment:
no areas of the wiki use the term "ruleset" (yet) except for tournament rule descriptions

```suggestion
| ![pp committee avatar](img/ppcommittee.png) | [@pp_committee](https://twitter.com/pp_committee) | Difficulty calculation announcements for all game modes, run by the [Performance Points Committee](/wiki/People/Performance_Points_Committee). |
```